packages/checkoutcom-integration/src/CheckoutcomCustomFormFields.test.tsx

```diff
@@ -8,28 +8,19 @@ import { EventEmitter } from 'events';
 import { Formik } from 'formik';
 import { noop } from 'lodash';
 import React, { FunctionComponent } from 'react';
-
+import {
```

Comment:
Since this test is still using enzyme, I will suggest we keep the suffix to be `spec` as compared to `test`. Since we want to keep `test` for RTL tests and use that for migrating tests.